scripts/genotypeQC.R

```diff
@@ -342,23 +557,23 @@ reldata1<-read.table(related.file, header = T)
 
 ##options for  plots Z1 vs Z0

```

Comment:
##options for  plots Z1 vs Z0
z1vz0<- ggplot(reldata1,aes(x=Z0,y=Z1))+
  stat_sum(aes(size = factor(..n..)), geom = "point")+
  scale_size_discrete(range = c(2, 8), name="Number of Samples")+
  theme_bw()+
  ggtitle("Sample relatedness")+
  theme(text=element_text(size=10,family="Helvetica"))+
  coord_cartesian(xlim = c(0, 1), ylim=c(0,0.6))


##pi_hat vs expected plot
obsvexp<- ggplot(reldata1,aes(x=PI_HAT,y=EZ))+
  geom_point()+
  theme_bw()+
  ggtitle("Relatednes coefficient")+
  theme(text=element_text(size=10,family="Helvetica"))+
  coord_cartesian(xlim = c(0, 1), ylim=c(0,1))+
  xlab("Observed R.C.")+
  ylab("Expected R.C.")


### saving out plots. 
relatedness.title <- paste0("Sample relatedness for austosomal and pseudo-autosomal chromosomes", 
                            "\n", opt$name, " ", date())
relatedness.file <- file.path(output, "06_relatedness.tiff")
#generating plots
tiff(relatedness.file,  
     width = 3000, height = 1500, 
     units = "px", res = 300, compression = "lzw")
grid.arrange(z1vz0, obsvexp, ncol=2, top= relatedness.title)
dev.off()